stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -558,7 +558,7 @@ public void onClose(Status status, Metadata trailers) {
     /**
      * Waits until there is a Runnable, then executes it and all queued Runnables after it.
      */
-    public void waitAndDrain() throws InterruptedException {
```

Comment:
Can we avoid these unrelated changes (here and above)? I actually put the public there on purpose, although I agree removing it works, too.